features/data_converter/empty/feature.java

```diff
@@ -0,0 +1,65 @@
+package data_converter.empty;
+
+import io.temporal.api.common.v1.Payload;
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+import io.temporal.failure.ApplicationFailure;
+import io.temporal.sdkfeatures.Feature;
+import io.temporal.sdkfeatures.Run;
+import io.temporal.sdkfeatures.Runner;
+import io.temporal.sdkfeatures.SimpleWorkflow;
+
+import java.time.Duration;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import com.google.protobuf.util.JsonFormat;
+
+@ActivityInterface
+public interface feature extends Feature, SimpleWorkflow {
+
+  @ActivityMethod
+  void activity(String input);
+
+  class Impl implements feature {
+    /**
+     * run a workflow that calls an activity with a null parameter.
+     */
+    @Override
+    public void workflow() {
+      var activities = activities(feature.class, builder -> builder
+              .setScheduleToCloseTimeout(Duration.ofMinutes(1)));
```

Comment:
We typically use `startToClose`, while schedule to close is fine in this specific case, it's not recommended in most cases where activities should retry indefinitely.
Once a schedule to close timeout is hit the activity will no longer be retried.

The reason I mention this is because this repo has some value as "samples" or "reference" repo where users might look up how the system behaves, we want people to copy our recommended best practices.

Reply:
will do